src/main/java/hudson/plugins/ansicolor/AnsiOutputStream.java

```diff
@@ -42,7 +42,7 @@
  */
 public class AnsiOutputStream extends FilterOutputStream {
 
-    public static final byte[] REST_CODE = resetCode();
+    private static final byte[] REST_CODE = resetCode();
```

Comment:
Yeah, that was my feeling too, we should really leave this alone. But there's an actual "UTF-8" change in there too that seems to fix something? Should it go upstream?